homeassistant/components/flexmeasures/helpers.py

```diff
@@ -0,0 +1,29 @@
+"""Helper to create S2 FlexMeasures Client instance."""
+from datetime import datetime
+
+from homeassistant import config_entries
+
+
+def get_from_option_or_config(key: str, entry: config_entries.ConfigEntry):
```

Comment:
How often are these used? I'd suggest putting them where they are used if its only one file. Personally helpers and utils files are in general are bad pattern.

Reply:
removed helpers.py and moved the functions to the correct files: 81ed50a29d91d1ecd71c489f258095868d2fd4ce